packages/reassure-compare/src/output/markdown.ts

```diff
@@ -39,9 +46,30 @@ async function writeToFile(filePath: string, content: string) {
   }
 }
 
+function buildMetadataMarkdown(name: string, metadata: MeasurementMetadata | null) {
+  if (metadata) {
+    if (metadata.branch && metadata.commitHash) {
+      return `**${name}**: \`${metadata.branch}\` (\`${metadata.commitHash}\`)`;
+    }
+
+    if (metadata.branch) {
+      return `**${name}**: \`${metadata.branch}\``;
+    }
+
+    if (metadata.commitHash) {
+      return `**${name}**: \`${metadata.commitHash}\``;
+    }
+  }
```

Comment:
nit: Using optional chaining should allow for flattening of `if`s
```suggestion
    if (metadata?.branch && metadata?.commitHash) {
      return `**${name}**: \`${metadata.branch}\` (\`${metadata.commitHash}\`)`;
    }

    if (metadata?.branch) {
      return `**${name}**: \`${metadata.branch}\``;
    }

    if (metadata?.commitHash) {
      return `**${name}**: \`${metadata.commitHash}\``;
    }
```